go/vumitools/app_worker.py

```diff
@@ -211,53 +212,25 @@ def _find_outboundmessage_for_event(self, event):
 
     _EVENT_OUTBOUND_CACHE_KEY = "outbound_message_json"
 
-    def _get_outbound_from_event_cache(
-            self, event, key=_EVENT_OUTBOUND_CACHE_KEY):
-        """ Retrieve outbound message from the cache on an event.
-
-        :type event:
-            TransportEvent
-        :param event:
-            Event to look up the outbound message on.
```

Comment:
Yes. They're internal methods that are each used exactly once and they don't really need pages of docstring. When I removed the `key` param I also removed the rest to reduce the amount of scrolling I had to do to read the code.